_stable/server/hello-world.md

```diff
@@ -126,6 +126,24 @@ async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
 
 To see all the snippets put together, check out the [full example][example]!
 
+This example uses the `http1` module to create a server that speaks HTTP/1.
```

Comment:
I'd move this to after the "Also, if `service_fn`"... and add a subheading, like `## HTTP/2`.